blog_post.md

```diff
@@ -0,0 +1,144 @@
+### GSoC 2016 Summary, Adding categorical data support
+
+Support for categorical data is important for any data analysis tool. This summer I added categorical data support to:
+- to easily analyze categorical data in Daru
+- visualize categorical data
+- support regression with categorical variable in Statsample and Statsample-GLM
+
+[Here's](https://summerofcode.withgoogle.com/archive/2016/projects/5356167010189312/) my project page.
+
+Lets talk about each of them in detail.
+
+#### Analyzing catgorical data with Daru
+
+Categorical data is now readily recognized by Daru and Daru has all the procedure to deal with it.
```

Comment:
Add a hyperlink to the daru homepage in the first instance of mentioning daru. Should be done whenver you mention a new gem so the user can quickly go to the respective page and see what you're talking about in further detail.